### PR TITLE
[doc] update README and remove unnecessary imports in setup.py

### PR DESCRIPTION
**Pull request by hezhizhen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,14 +23,17 @@ ExpMan provides a set of handy commands that not only allows beginners to quickl
...
-```bash
-git clone https://github.com/megvii-research/expman $HOME/.expman
-```
-3. Load expman by adding one line in `~/.zshrc`
-```bash
-echo '[[ -o interactive ]] && source $HOME/.expman/expman.zsh' >> ~/.zshrc
-```
+    ```bash
+    brew install fzf
+    ```
```
Comment (bigeagle) [inline README.md]: I think it's better to keep blank here. The correct method of installation depends on user's platform. It's difficult to cover. Fzf's doc already include various installation methods.

**Review by bigeagle — CHANGES_REQUESTED** [review]
Great PR :)

File: README.md (hunk 1 above)
Comment (hezhizhen) [reply README.md]: You're right. I'll remove it.

Comment (hezhizhen) [reply README.md]: Done. PTAL @bigeagle

Comment (bigeagle) [reply README.md]: LGTM 👍

**Pull request merged**